R/utils.R

```diff
@@ -20,7 +20,16 @@ base_pkgs <- function() {
 }
 
 #' @importFrom pkgcache ppm_snapshots
-get_ppm_snapshot_by_date <- function(date) {
+#' @examplesIf Sys.getenv("R_USER_CACHE_DIR", "") != ""
+#' get_ppm_snapshot_by_date(NA)
+#' get_ppm_snapshot_by_date("2023-08-01")
+#' get_ppm_snapshot_by_date(Sys.Date() + 10)
+get_ppm_snapshot_by_date <- function(date,
+                                     fallback_repo = file.path(pkgcache::ppm_repo_url(), "latest")) {
```

Comment:
do we ever use a `fallback_repo` in the codebase?
YAGNI?

Reply:
I mean we use a fallback functionality but I guess making a new function parameter out of this is probably too much

Reply:
Done in 7ce8e09